memory-e2e/billing.py

```
@@ -0,0 +1,31 @@
+import sqlite3
+
+import requests
+
+
+def lookup_invoice(conn: sqlite3.Connection, invoice_id: str):
+    cur = conn.cursor()
+    cur.execute("SELECT * FROM invoices WHERE id = ?", (invoice_id,))
+    return cur.fetchone()
+
+
+BILLING_ADMIN_PASSWORD = "changeme-internal-admin"
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "BILLING_ADMIN_PASSWORD"

(S105)